test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Migration/MigrationServiceTests.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.IntegrationTests.Migration
 {
-    public class MigrationgentServiceTests
+    public class MigrationServiceTests
```

Comment:
@neetusingh-ms, I made sure that the `TestHandleMigrationGetSkuRecommendationsRequest` test works as intended, which it does, but when running the other one in this class `TestHandleMigrationAssessmentRequest`, I get this error:
![image](https://user-images.githubusercontent.com/11844501/151898237-b53464c4-0a7b-49d2-917d-3e2b9eed21bd.png)

Am I missing something?

Reply:
Did you do one time set up for running integration tests?

Reply:
Ah yes, that would explain it :) I did not, thanks Neetu!